### PR TITLE
Add Circle. Modify origami.json. Remove html, config.json & Travis

### PR DESCRIPTION
**Pull request by j-mes**
- Check for correct support link
- Change from Travis to Circle CI
- Move variables from /src/demos/config.json into origami.json
- Remove html files from /src/demos/
- Remove /src/demos/config.json
- Remove travis.yml
- Add circle.yml
- Passed OBT verify check
- Passed OBT runServer check

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -4,36 +4,49 @@
     "origamiVersion": 1,
     "support": "https://github.com/Financial-Times/o-promobox/issues",
     "supportStatus": "active",
+    "ci": {
+      "circle": "https://api.travis-ci.org/repos/Financial-Times/o-overlay/builds.json"
```
Comment (alicebartlett) [inline origami.json]: This should be https://api.travis-ci.org/repos/Financial-Times/o-promobox/builds.json

File: origami.json [hunk 2]
```diff
@@ -4,36 +4,49 @@
     "origamiVersion": 1,
```
Comment (onishiweb) [inline origami.json]: Indenting is a bit all over the place in here

**Comment by onishiweb** [comment]
All good, just need to sort the indenting in the origami.json

**Comment by AlbertoElias** [comment]
Why is this a do not merge?

**Comment by j-mes** [comment]
Passed OBT verify but failed OBT runServer and I can't see where it's breaking.

**Comment by AlbertoElias** [comment]
But it passed circle

File: origami.json [hunk 3]
```diff
@@ -1,39 +1,52 @@
...
+		},
+		{
+			"name": "standard-skinny",
+			"template": "demos/src/standard-skinny.mustache",
+			"expanded": false,
+			"description": ""
+		},
+		{
+			"name": "custom-responsive",
+			"template": "demos/src/custom-responsive.mustache",
```
Comment (AlbertoElias) [inline origami.json]: the template is called custom-responsivity.mustache, feel free to rename it to custom-responsive

Comment (j-mes) [inline origami.json]: Fixed.

**Comment by onishiweb** [comment]
Just tested this locally and it's all working fine. Happy for you to merge this when ready.

**Pull request merged**